perk-store-webapps/src/main/webapp/vue-app/components/perk-store/OrdersFilter.vue

```diff
@@ -15,78 +15,74 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <v-navigation-drawer 
-    v-show="display"
-    v-model="display"
-    class="elevation-3 productOrderFilter"
-    temporary
-    absolute
-    width="380"
-    :right="rightDrawer"
-    :left="leftDrawer">
-    <v-toolbar color="transparent">
-      <v-toolbar-title class="text-truncate">{{ $t('exoplatform.perkstore.label.filterOrders') }}</v-toolbar-title>
-    </v-toolbar>
-    <v-divider />
-    <v-container grid-list-xl class="border-box-sizing pe-1 productOrderFilterContent">
-      <v-layout wrap column>
-        <h4><v-checkbox v-model="filter.searchInDates" :label="$t('exoplatform.perkstore.label.searchInDates')" /></h4>
-        <template v-if="filter.searchInDates">
-          <v-divider />
-          <v-date-picker
-            v-model="selectedDate"
-            type="date"
-            class="border-box-sizing" />
-        </template>
-        <v-checkbox v-model="filter.notProcessed" :label="$t('exoplatform.perkstore.label.notCompletelyProcessed')" />
-        <template v-if="!filter.notProcessed">
-          <v-checkbox
-            v-model="filter.ordered"
-            :label="$t('exoplatform.perkstore.label.status.ordered')" />
-          <v-checkbox
-            v-model="filter.canceled"
-            :label="$t('exoplatform.perkstore.label.status.canceled')" />
-          <v-checkbox
-            v-model="filter.error"
-            :label="$t('exoplatform.perkstore.label.status.error')" />
-          <v-checkbox
-            v-model="filter.paid"
-            :label="$t('exoplatform.perkstore.label.status.paid')" />
-          <v-checkbox
-            v-model="filter.partial"
-            :label="$t('exoplatform.perkstore.label.status.partial')" />
-          <v-checkbox
-            v-model="filter.delivered"
-            :label="$t('exoplatform.perkstore.label.status.delivered')" />
-          <v-checkbox
-            v-model="filter.refunded"
-            :label="$t('exoplatform.perkstore.label.status.refunded')" />
-          <v-checkbox
-            v-model="filter.fraud"
-            :label="$t('exoplatform.perkstore.label.status.fraud')" />
-        </template>
-      </v-layout>
-    </v-container>
-    <template v-slot:append>
-      <v-divider />
-      <v-footer color="transparent" class="px-8 mb-2">
+  <exo-drawer
+    ref="ordersFilterDrawer"
+    class="productOrderFilter"
+    :right="!$vuetify.rtl"
+    width="380">
```

Comment:
Is there a need to set the width at this level ?

Reply:
checked and no need to set it! removed!